test/test_torch.py

```diff
@@ -5414,6 +5414,80 @@ def test_multinomial_empty_wo_replacement(self, device):
         self._test_multinomial_empty(device, False, 1)
         self._test_multinomial_empty(device, False, 2)
 
+    @onlyCPU
+    @dtypes(torch.float, torch.double)
+    def test_grad_scaling_unscale(self, device, dtype):
```

Comment:
Does CUDA have similar tests? If so, please explain why we cannot share.

Reply:
`test_grad_scaling_unscale` is also defined in test_cuda_multigpu.py. Will move it here since GradScaler for CPU is not added yet.